### ISSUE TITLE
Using QR Code Styling with Vanilla Javscript : possible ?

### ISSUE THREAD

**Issue by smappio** [kozakdenys/qr-code-styling#131]
Hello,

Sorry for the question if it makes non sense, but .. is there a way to use this library with Vanilla JS ? 
Many thanks :)

**Comment by smappio** [comment]
Yeah, in fact I can't use anything else than html / Javascript as my web app is hosted on a third party framework, I can't take benefits of nom.

Using to integrate this lib, I get an error for the instanciation of a new qrcode object : QRCodeStyling undefined.

The issue is that I can't do an import of the QRCodeStyling object using pure Javascript script, right ?

I really hope I'll find a way to integrate and use your lib, it's awesome, really awesome, the generated qrcode are so nice !

**Comment by chiptu** [comment]
Maybe you can import vuejs with a cdn https://fr.vuejs.org/v2/guide/installation.html#CDN

**Comment by smappio** [comment]
Many thanks @chiptu  ! I have to test, not sure what thé impact of importing vuejs could be as I'm running on Bubble as a backend. I have to give a try ! Thanks !

**Comment by smappio** [comment]
> Maybe you can import vuejs with a cdn https://fr.vuejs.org/v2/guide/installation.html#CDN

Hello,

I tried but no way to get it working
So, the lib via a CDN would be, I guess, the only way to use it.

Is there any plan for accessibility via cdn ?

Kind regards
Chris
